docs/integrations/gitlab/discovery.md

```diff
@@ -47,6 +47,9 @@ of your backend.
  ): Promise<Router> {
    const builder = await CatalogBuilder.create(env);
 +  builder.addProcessor(
-+    GitLabDiscoveryProcessor.fromConfig(env.config, { logger: env.logger })
++    GitLabDiscoveryProcessor.fromConfig(env.config, {
++       logger: env.logger,
++       skipReposWithoutExactFileMatch: true,
```

Comment:
Hmm, I feel we should just mentioned the option rather than making it part of the default installation tbh

Reply:
Is it enough to change this setting to false? Or how do you see I should change this?

Reply:
I'm thinking we could keep the example as is, and then we could add a sentence in the text to the effect of "If you want to ... you can set the `skipReposWithoutExactFileMatch` option"

Reply:
Done, please have a look